gocourse03/drugsTask/go.mod

```
@@ -0,0 +1,5 @@
+module github.com/sarff/prjctr-golang_beginning/gocourse02/drugs_task
```

Папка називається `gocourse03/drugsTask`, тому і модуль має бути `gocourse03/drugsTask`.

Це тут написано https://go.dev/ref/mod#module-path

```suggestion
module github.com/sarff/prjctr-golang_beginning/gocourse03/drugsTask
```

---

Те саме стосується `mouseMetropolis`.

---

Може перейменувати `drugsTask` на просто `drugs`? По суті, у нас кожна директорія є таскою)